src/main/java/com/blockout/PaneParams.java

```diff
@@ -1,5 +1,6 @@
 package com.blockout;
 
+import com.minecolonies.util.Log;
```

Comment:
I'm not a fan of us using minecolonies classes inside of blockout. Blockout should have its own logger.